components/dash-core-components/tests/integration/markdown/test_markdown.py

```diff
@@ -90,6 +91,278 @@ def test_mkdw002_dcclink(dash_dcc):
     )
 
     dash_dcc.start_server(app)
-    dash_dcc.percy_snapshot("mkdw002 - markdowns display")
+    assert dash_dcc.get_logs() == []
+
+
+@pytest.mark.parametrize("is_eager", [True, False])
+def test_mkdw003_without_mathjax(dash_dcc, is_eager):
+    app = Dash(__name__, eager_loading=is_eager)
+
+    app.layout = html.Div(
+        [
+            dcc.Markdown("# No MathJax: Apple: $2, Orange: $3"),
+        ]
+    )
+
+    dash_dcc.start_server(app)
+    dash_dcc.wait_for_text_to_equal("h1", "No MathJax: Apple: $2, Orange: $3")
+    assert not dash_dcc.driver.execute_script("return !!window.MathJax")
+    assert dash_dcc.get_logs() == []
+
+
+@pytest.mark.parametrize("is_eager", [True, False])
+def test_mkdw004_inline_mathjax(dash_dcc, is_eager):
+    app = Dash(__name__, eager_loading=is_eager, assets_folder="../../assets")
+
+    app.layout = html.Div(
+        [
+            dcc.Markdown("# h1 tag with inline MathJax: $E=mc^2$", mathjax=True),
+        ]
+    )
+
+    dash_dcc.start_server(app)
+    assert dash_dcc.get_logs() == []
+
+
+@pytest.mark.parametrize("is_eager", [True, False])
+def test_mkdw005_block_mathjax(dash_dcc, is_eager):
+    app = Dash(__name__, eager_loading=is_eager, assets_folder="../../assets")
+
+    app.layout = html.Div(
+        [
+            dcc.Markdown(
+                """
+                    ## h2 tag with MathJax block:
+                    $$
+                    \\frac{1}{(\\sqrt{\\phi \\sqrt{5}}-\\phi) e^{\\frac25 \\pi}} =
+                    1+\\frac{e^{-2\\pi}} {1+\\frac{e^{-4\\pi}} {1+\\frac{e^{-6\\pi}}
+                    {1+\\frac{e^{-8\\pi}} {1+\\ldots} } } }
+                    $$
+                    ## Next line.
+                """,
+                mathjax=True,
+            ),
+        ]
+    )
+
+    dash_dcc.start_server(app)
+    assert dash_dcc.get_logs() == []
```

Comment:
Here it looks like the math isn't in the `<h2>` but we can still verify that it gets rendered, maybe give the markdown an ID like `"md"` then
```py
dash_dcc.wait_for_element("#md svg")
```
(just to make sure we aren't catching some other weird SVG on the page...)

Reply:
Addressed in c61b771f2e31b5f7165b0c866695a0f8ee586e57.